urdf/package.xml

```diff
@@ -28,15 +28,16 @@
   <build_depend>urdf_parser_plugin</build_depend>
   <build_depend>pluginlib</build_depend>
   <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
   <build_depend>tinyxml</build_depend>
 
-  <run_depend>liburdfdom-dev</run_depend>
-  <run_depend>liburdfdom-headers-dev</run_depend>
-  <run_depend>rosconsole_bridge</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf_parser_plugin</run_depend>
-  <run_depend>pluginlib</run_depend>
-  <run_depend>tinyxml</run_depend>
+  <exec_depend>liburdfdom-dev</exec_depend>
+  <exec_depend>liburdfdom-headers-dev</exec_depend>
```

Comment:
The latest package for `liburdfdom-headers-dev` on xenial has nothing but headers, so I don't think it needs to be an `<exec_depend>`. 

I'm surprised `liburdfdom-dev` is a run time dependency, but I don't see any non dev version in ubuntu and the latest package released to xenial does appear to have shared libraries in it.